package.json

```diff
@@ -36,8 +37,8 @@
     "svelte-check": "^3.5.0",
     "tslib": "^2.6.1",
     "typescript": "^5.1.6",
-    "vite": "^4.4.9",
-    "vitest": "^0.34.1"
+    "vite": "^5.1.6",
```

Comment:
I had to update vite to use some testing features, which led to a chain reaction of updating everything. Lets be sure to do some manual regression tests in preview and when deployed